### ISSUE TITLE
[WARNING] This project is dead, time to move on to Wire or something.

### ISSUE THREAD

**Issue by ghost** [ricochet-im/ricochet#578]
To be honest @s-rah and @special , no one can verify you guys are working on this project or not.
Did you abandoned this software? It seems so.

Almost 2 years for no release.
Why not just merge current pull requests, update tor binary to latest, pack it and release it as "maintenance release" already?
If you're working on ricochet-go, it will sure take time because it is currently in development from scratch.

But we - users - can't wait too long. Another year? Two year? Three years? Come on.

TLDR, please release latest Ricochet. Just merge pull requests and update tor. Easy enough.
If you don't do this, we should find an alternative.

**Comment by ghost** [comment]
And it would be nice if you post a status every month, IF you're still working on this.

**Comment by ilovelinux** [comment]
It's not dead. Take a look at https://github.com/ricochet-im/ricochet/issues/574, https://github.com/ricochet-im/ricochet/issues/568, https://github.com/ricochet-im/ricochet/issues/555#issuecomment-324094653

**Comment by ghost** [comment]
@ilovelinux You're blind and don't want to face the truth. Read my original post already.

**Comment by ghost** [comment]
https://github.com/ricochet-im/ricochet/issues/579

**Comment by s-rah** [comment]
Since I've been pinged multiple times to comment on this, but I will state I am speaking for myself here.

There are a number of reasons I've been quiet, one of the main ones is that I've been in hospital dealing with medical stuff. Life gets in the way sometimes  ¯\_(ツ)_/¯

However, https://github.com/s-rah/go-ricochet is actively being developed (last commit 20 days ago, last activity today). Ricochet as a concept and protocol is far from dead.

The reason go-ricochet is actively being developed is because of the other main reason I've been quiet - I've been building an organization, Open Privacy (https://openprivacy.ca), our mandate is to conduct research and build privacy enhancing tools - because building tools like ricochet is a lot of work, requires time and money and people - and very few people honestly want to fund that in a sustainable way. My entire job now revolves around building a future where everyone can benefit from a tool like ricochet. So even though it seems like there is no work going on, I can assure you there is a multitude of work going on.

Open Privacy is funding the development of Cwtch, an extension of the ricochet protocol to add a bunch of stuff like offline sending, group messaging etc. A prototype and a paper exists, are semi-public and we will be more widely circulating them very soon. Research and development takes time but there is money and development effort behind it. There is also budget and roadmap planning for an initial android application towards Winter 2018. Support towards that effort is always welcome (https://openprivacy.ca/donate/) 

As for this project, there is clearly some maintenance to be done to prevent longer-term rot. Standard ricochet packagings  e.g. debian should , AFAICR, prefer system installed/newer Tor versions. In my opinion, there is no reason in the short term and in many cases to consider Ricochet unsafe to use,but it is experimental and volunteer driven.

I agree though that it would be nice to get that into an official release with some of the small bug fixes in PR, so that standalone versions are a little more resilient. 

That kind of work takes time, effort and a person, and there is a balance between other work (that actually pays bills), the future, and other life to deal with.  I still think Ricochet as a tool is useful, and will remain useful in the mid-term and personally I think labeling it as **Discontinued** or **Dead** is premature, but I will leave that for @special to comment on.

**Comment by ghost** [comment]
@s-rah Would you mind posting this URL to your Twitter for identity verification? Your Github might be hacked by @ilovelinux.

**Comment by ohmynameisrico** [comment]
@s-rah And one more thing: Why you guys(either) told the Tor Project that the project is discontinued?

**Comment by ohmynameisrico** [comment]
Ok, let's continue in #579 ok?

**Comment by smaragdus** [comment]
[Ricochet](https://github.com/ricochet-im/ricochet) is dead, dead as a dodo.
